lib/rawHttpRequest/processBody.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
+
+const _ = require('lodash');
+
+//Since there may be multiple files, need to download data for all of them and then set the body
+function getFile ({ file, mime_type }) {
+	//This is a format needle accepts without having to read in the file in order to upload in the request
+	return {
+		file,
+		content_type: mime_type,
+	};
+}
+//This functionality is derived from the HTTP client connector
+
+module.exports = async function (body) {
+
+	const bodyType = _.keysIn(body)[0];
```

Comment:
can body have multiple properties? the order of returned keys is not guaranteed, so [0] seems questionable

Reply:
No - the schema is a oneOf, so there should only ever be on property.